--- a/.jenkins/Jenkinsfile.snapshot
+++ b/.jenkins/Jenkinsfile.snapshot
@@ -4,7 +4,7 @@
 pipeline {
     agent {
         docker {
-            image "registry.firstdark.dev/java21:latest"
+            image "registry.firstdark.dev/java17:latest"
             alwaysPull true
         }
     }
@@ -13,7 +13,7 @@
         stage("Notify Discord") {
             steps {
                 discordSend webhookURL: env.SSS_WEBHOOK,
-                        title: "Deploy Started: ${projectName} Port Deploy #${BUILD_NUMBER}",
+                        title: "Deploy Started: ${projectName} 1.20.4 Deploy #${BUILD_NUMBER}",
                         link: env.BUILD_URL,
                         result: 'SUCCESS',
                         description: "Build: [${BUILD_NUMBER}](${env.BUILD_URL})"
@@ -52,7 +52,7 @@
                 projectName: "${projectName}",
                 projectIcon: "${projectIcon}",
                 modLoaders: "forge|neoforge|fabric|quilt",
-                minecraftVersions: "1.20.6",
+                minecraftVersions: "1.20.4",
                 failWebhook: env.SSS_WEBHOOK,
                 publishWebhooks: "${env.SSS_WEBHOOK}|${env.FDD_WH}"
 
